smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -174,9 +174,17 @@ public Task Install()
                 300); // 5 min timeout because install can be slow on raspberry pi
         }
 
-        public async Task Configure(DeviceProvisioningMethod method, string hostname, string deviceCaCert, string deviceCaPk, string deviceCaCerts, LogLevel runtimeLogLevel)
+        public async Task Configure(DeviceProvisioningMethod method, Option<string> agentImage, string hostname, string deviceCaCert, string deviceCaPk, string deviceCaCerts, LogLevel runtimeLogLevel)
         {
-            Console.WriteLine($"Setting up iotedged with agent image 1.0");
+            agentImage.ForEach(
```

Comment:
is it better to combine this ForEach with ForEach on line 232?

Reply:
I thought about this too, but liked this executing at the very beginning.

Reply:
nit: should put together; will make sure any updates will not miss the other?  your call.

Reply:
My only concern is the unhappy path where something fails. If we move the log statement down then we will see less information in this error case.